Source/Mosa.UnitTests.x86/Mosa.UnitTests.x86.csproj

```diff
@@ -1,4 +1,23 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
+	<PropertyGroup>
```

Comment:
merge issue, all these "new" lines should be removed

Reply:
Check.